pubsub/drivertest/drivertest.go

```diff
@@ -141,6 +142,7 @@ func (verifyAsFailsOnNil) MessageCheck(m *pubsub.Message) error {
 func RunConformanceTests(t *testing.T, newHarness HarnessMaker, asTests []AsTest) {
 	tests := map[string]func(t *testing.T, newHarness HarnessMaker){
 		"TestSendReceive":                                         testSendReceive,
+		"TestReceiveSend":                                         testReceiveSend,
```

Comment:
Don't you need to add golden files for existing providers?

Reply:
I did for gcp but the aws driver hangs on this.

Reply:
OK... I think you need to fix that before you can submit this. The tests for the PR fail.

Reply:
Looking into it.

Reply:
Hmm, I think this has to do with the queues not getting auto-subscribed to the topics in the tests.

Reply:
Done.